codecademy/taking_a_vacation.py

```diff
@@ -35,11 +35,15 @@ def rental_car_cost(days):
 
 def trip_cost(city, days, spending_money):
     """calculate total sum"""
-    assert type(spending_money) == int and spending_money > 0, "{} was not int or positive integer > 0".format(spending_money)
+    assert type(spending_money) == (int or float) and spending_money >= 0,\
```

Comment:
I think you wanted to write this:
```python
    assert type(spending_money) in (int, float) and spending_money >= 0
```
Note: `(int or float)` looks like a boolean statement but will actually return `int` ... please only use `or` when you have a boolean on the left and on the right. Like:
```python
if 1 > 0 or 2 > 1:
  print('yay!')
```